frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -110,6 +110,11 @@ export const FormComponentConfig = ({
     ? t('ux_editor.component_other_properties_hide_many_settings')
     : t('ux_editor.component_other_properties_show_many_settings');
 
+  const hideBooleanKeyHelpText = (propertyKey: string) => {
+    const isFirstBooleanKey = propertyKey === booleanPropertyKeys[0];
+    return isFirstBooleanKey ? t('ux_editor.component_properties.hidden-help-text') : undefined;
+  };
+
```

Comment:
I think you should use the `componentPropertyHelpText` function in EditBooleanValue instead of making own logic for adding help text to the first boolean key.

Reply:
did not work by using hook , I am not sure but maybe because we still need to add the new norwegian  text? because we have just english before.

Reply:
The hook is used to lookup our own texts in the nb file. So the text key for help text in the first boolean switch needs to be adjusted to the same variant as the hook is looking for.